### ISSUE TITLE
Peer Review by Parth 

### ISSUE THREAD

**Issue by samantparth** [davidpxrk/childrens-book-rating#1]
**Summary**

This repo analyses the relationship between different characteristics of children's books and aims to identify how much these characteristics influences the books' ratings. This was done using a bayesian logistic regression model that had variables including the cover, number of pages, year it was published, and total number of reviews that predict whether the rating was high.

**Strong positive points**:

- The introduction is quite captivating and shows the importance of the topic well
- The predictors of your model are well-chosen and could potentially be a good predictor of how whether the book gets a high rating.
- Great job setting up a foundation for which graphs will be used in the paper.

 **Critical improvements needed:**
- Make sure that the preambles for the scripts are updated so that they match the file in question
- Make sure you have a file that downloads the dataset
- more information could be provided about the significance of the findings that you will have. For instance, you could say that these results can push the boundaries of literature by highlighting what makes a captivating and highly-rated book.

 **Suggestions for improvement:**
- In the paragraph for the estimand, it may be better to include details of the model in other sections.
- I would reword the part that mentions that a high rating is connected to whether educators/parents will give their book to children, as there are likely more important reasons for a high rating. Maybe you could, instead, say that the rating scale ensures quality control of the books that one reads as well as how well it can captivate the reader.
- Assuming it is not a temporary file, it may be better to remove the `paper_files/figure_pdf` folder.
- the paper.pdf file contains many datasets – including some that appear to be cut off. Make sure that this is addressed (although I am sure you are aware of this).
- There are some grammar mistakes, including some in the abstract. Make sure to proofread before submitting.

 **Evaluation:**
R is appropriately cited (1 pts): 
0/1: R is not appropriately cited in the bibliography and in the paper

Data  is appropriately cited (1 pts):
0/1: data is not appropriately cited in the bibliography and in the paper

Class paper (1 pts):
1/1: No indication this is a class project

LLM usage documented (1 pts):
0/1: LLM usage has not been updated/mentioned

Title (2 pts):
0/2: Not started

Author, date, and repo (2 pts):
2/2: The author, date, and GitHub repo link are included.

Abstract (4 pts):
3.5/4: Overall quite good, but could probably reword the last sentence on significance

Introduction (4 pts):
1/4: The introduction includes a background as well as its significance. Make sure to also include a gap that needs to be filled, what was done/found, as well as the structure of the paper.

Estimand (1 pts):
1/1: The estimand is mentioned and explained upon.

Data (10 pts):
0/10: Not started

Measurement (4 pts):
0/4: Not started

Model (10 pts):
0/10: Not started

Results (10 pts):
0/10: Not started

Discussion (10 pts):
0/10: Not started

Prose (6 pts):
3/6: Overall okay prose, but make sure to fix grammar mistakes.

Cross-references (1 pts):
0/1: Cross-references not included

Captions (2 pts):
0/2: Not included

Graphs/tables/etc (4 pts):
3/4: Graphs/tables exist but some a little bit tough to read, including the "Count of Hardcover and Paperback by Rated High" graph.

Surveys, sampling, and observational data appendix (10 pts):
0/10: Not Started

Referencing (4 pts):
0/4: Not Started

Commits (2 pts):
2/2: Handful of commits that give information about the paper.

Sketches (2 pts):
0/2: Not Started

Simulation (4 pts):
0/4: Not Started

Tests (4 pts):
0/4: Not Started

Parquet (1 pts):
1/1: raw data and analysis data are both present.

Reproducible workflow (4 pts):
4/4: Workflow is reproducible and easy to create.

Miscellaneous (3 pts):
0/3

Enhancements (4 pts):
0/4: Not started

**Estimated overall mark:**
21.5/112.

**Other Comments** 
The main reason for the low mark is because most of the paper hasn't been done yet. Otherwise, I think your topic is quite interesting and think this is a great start.

I hope some of my feedback can help, and good luck with the rest of this paper! :)

**Comment by davidpxrk** [comment]
Hi Parth, thanks for your feedback! I added most of the missing sections and deleted some unnecessary files as mentioned.
